### PR TITLE
DC-3260 - welsh support

### PR DESCRIPTION
**Pull request by elabeca**

### CODE REVIEW

File: test/controllers/MessageControllerSpec.scala [hunk 1]
```diff
@@ -60,9 +62,8 @@ class MessageControllerSpec extends PlaySpec with GuiceOneAppPerSuite with MockA
 
       private val messagesContent = controller.messagePartial(messages).toString()
 
-      messagesContent must include("this on 19 February 2021 at 10:29am")
-      messagesContent must include("on 1 March 2021 at 10:29am")
-      messagesContent must include("senderName sent")
+      messagesContent must include("senderName sent this on 19 February 2021 at 10:29am")
```
Comment (muralishub) [inline test/controllers/MessageControllerSpec.scala:65]: should senderName be bold ?

File: test/controllers/MessageControllerSpec.scala [hunk 2]
```diff
@@ -76,9 +77,8 @@ class MessageControllerSpec extends PlaySpec with GuiceOneAppPerSuite with MockA
 
       private val messagesContent = controller.messagePartial(messages).toString()
 
-      messagesContent must include("this on 19 February 2021 at 10:29am")
-      messagesContent must include("on 1 March 2021 at 10:29am")
-      messagesContent must include("You sent")
+      messagesContent must include("You sent this on 19 February 2021 at 10:29am")
```
Comment (muralishub) [inline test/controllers/MessageControllerSpec.scala:80]: you sent should be bold?

File: test/views/helpers/HtmlUtilSpec.scala [hunk 3]
```diff
@@ -16,28 +16,25 @@
...
 
 @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
 class HtmlUtilSpec extends PlaySpec {
+  val messagesApi = new DefaultMessagesApi()
+  implicit val messages = MessagesImpl(Lang("en"), messagesApi)
 
-  "conversation readableTime function returns correct readable timestamp" in {
-    readableTime(DateTime.parse("2021-02-19T10:29:47.275Z")) must be("19 February 2021 at 10:29am")
+  "conversation readableTime function returns correct readable timestamp in English" in {
+    readableTime(DateTime.parse("2021-02-19T10:29:47.275Z")) must be("19 February 2021 at 10:29AM")
```
Comment (muralishub) [inline test/views/helpers/HtmlUtilSpec.scala:32]: Are we changing am to AM ? accesibility ?

**Review by muralishub — APPROVED** [review]

**Review by masahmad — APPROVED** [review]

File: test/views/partials/conversationInboxSpec.scala [hunk 4]
```diff
@@ -52,7 +54,29 @@ class conversationInboxSpec extends TemplateUnitSpec[MessageInbox] {
...
+
+    val messagesApi: MessagesApi = new DefaultMessagesApi(
+      messages = Map(
+        "en" -> messagesResourceEn,
+        "cy" -> messagesResourceCy
+      )
+    )
+
+    val messagesEn = MessagesImpl(Lang("en"), messagesApi)
+//    val messagesCy = MessagesImpl(Lang("cy"), messagesApi)
```
Comment (muralishub) [inline test/views/partials/conversationInboxSpec.scala:78]: we can get rid of this comment

File: test/views/partials/messageContentSpec.scala [hunk 5]
```diff
@@ -16,46 +16,77 @@
...
+      val messageContent =
+        new messageContent(layout)(
+          MessageView(Some("Mike"), DateTime.parse("2021-02-19T10:29:47.275Z"), None, "message body"))(
+          messagesCy,
+          requestHeader).toString
 
-      messageContent must include("Mike")
-      messageContent must include("sent text")
-      messageContent must not include ("First read")
+      messageContent must include("Mike wnaeth anfon y neges hon ar 19 Chwefror 2021 am")
```
Comment (muralishub) [inline test/views/partials/messageContentSpec.scala:81]: Do we have space before am for welsh ?

Comment (elabeca) [reply test/views/partials/messageContentSpec.scala:81]: This "am" as in "at" but in Welsh, not "am" as "AM/PM"

**Pull request merged**